tests/providers/amazon/aws/operators/test_datasync.py

```diff
@@ -710,7 +710,7 @@ def test_killed_task(self, mock_wait, mock_get_conn):
         # ### Begin tests:
 
         # Kill the task when doing wait_for_task_execution
-        def kill_task(*args):
+        def kill_task(*args, **kwargs):
```

Comment:
What is this change for? It doesn’t seem to be related without others.

Reply:
Thanks for the feedback, yes it is needed. 

Without it, the test fails:
`E               TypeError: kill_task() got an unexpected keyword argument 'max_iterations'`

We are mocking `AWSDataSyncHook::wait_for_task_execution` which takes the kwarg `max_iterations` (unchanged)
On `AWSDataSyncOperator` line 364 we are passing this kwarg for the first time, so the mock needs to be able to receive it also. (changed)